io/include/gz/common/DataFrame.hh

```diff
@@ -0,0 +1,201 @@
+/*
+ * Copyright (C) 2022 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+#ifndef GZ_COMMON_DATAFRAME_HH_
+#define GZ_COMMON_DATAFRAME_HH_
+
+#include <array>
+#include <numeric>
+#include <sstream>
+#include <string>
+#include <stdexcept>
+#include <unordered_map>
+#include <vector>
+
+#include <gz/common/CSVStreams.hh>
+#include <gz/common/Io.hh>
+
+#include <gz/math/TimeVaryingVolumetricGrid.hh>
+
+namespace gz
+{
+  namespace common
+  {
+    /// \brief An abstract data frame.
+    ///
+    /// \tparam K Column key type
+    /// \tparam V Column value type
+    template <typename K, typename V>
+    class DataFrame
+    {
+      /// \brief Check if column key is present.
+      public: bool Has(const K &_key) const
```

Comment:
Document `_key` and return value.

Reply:
Done in https://github.com/gazebosim/gz-common/pull/402/commits/9a3c84d6d1d6d76a62b35fd9ab04d31a0ca8178b.